PoetAssistant/src/AppDelegate.swift

```diff
@@ -36,12 +37,50 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 		Settings.getTheme().apply()
 		return true
 	}
+	
+	func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
+		guard let sendingAppID = options[.sourceApplication] as? String else {
```

Comment:
We don't need to do any other init here? like everything we do inside the `didFinishLaunchingWithOptions` function?